### PR TITLE
fix the ray version and the script

### PR DESCRIPTION
**Pull request by AmeerHajAli**

### CODE REVIEW

File: driver.py [hunk 1]
```diff
@@ -10,13 +10,11 @@
...
 # uses an Anyscale cluster to run Ray.
-os.environ["RAY_ADDRESS"]="anyscale://onboarding"
-ray.client().connect()
+ray.init()
 
 N = 200
 print(f"Getting the {N}th fibonnaci number for you, a few times")
-obj = [run_job.remote(N) for i in range(5)]
-ray.get(obj)
+ray.get([run_job.remote(N) for i in range(5)])
```
Comment (grechaw) [inline driver.py:17]: I like to leave obj in to have a chance to explain object references.

File: driver.py [hunk 2]
```diff
@@ -10,13 +10,11 @@
 # to connect to a ray cluster.
 # If RAY_ADDRESS begins with "anyscale://" then your program 
 # uses an Anyscale cluster to run Ray.
-os.environ["RAY_ADDRESS"]="anyscale://onboarding"
```
Comment (grechaw) [inline driver.py:13]: Ah this is what @pcmoritz was referring to before.  Makes sense to remove it here.

**Review by grechaw — APPROVED** [review]
I don't mind removing the obj, it's cleaner looking.  We'll rely on other materials to for tutorial and training.

**Comment by AmeerHajAli** [comment]
I might have accidentally broke something, can you please verify if this onboarding flow works for you before I can merge it? Ideally we would have a test for it.

**Comment by AmeerHajAli** [comment]
I tested that it works.

**Pull request merged**